internal/pkg/service/buffer/storage/local/reader/reader.go

```diff
@@ -0,0 +1,174 @@
+// Package reader provides reading of tabular data from local storage for upload to staging storage.
+// Data may be compressed on-tly-fly according to the configuration.
+package reader
```

Comment:
Tu pridavam podporu citania dat z local storage, pocas uploadu do staging storage.
Data mozeme volitelne este komprimovat, aj ked to nebude asi preferovane/najrychlejsie.